sonaranalyzer-dotnet/its/expected/akka.net/Akka.Cluster-{6AB00F61-269A-4501-B06A-026707F000A7}-S4136.json

```diff
@@ -0,0 +1,100 @@
+{
+"issues":  [
+{
+"id":  "S4136",
+"message":  "All 'Equals' signatures should be adjacent",
+"location":  [
+{
+"uri":  "sources\akka.net\src\core\Akka.Cluster\ClusterMetricsCollector.cs",
+"region":  {
+"startLine":  270,
+"startColumn":  22,
+"endLine":  270,
+"endColumn":  28
+}
+},
+{
+"uri":  "sources\akka.net\src\core\Akka.Cluster\ClusterMetricsCollector.cs",
+"region":  {
+"startLine":  280,
+"startColumn":  30,
+"endLine":  280,
+"endColumn":  36
+}
+}
+]
+},
+{
+"id":  "S4136",
+"message":  "All 'Equals' signatures should be adjacent",
+"location":  [
+{
+"uri":  "sources\akka.net\src\core\Akka.Cluster\ClusterMetricsCollector.cs",
```

Comment:
this is a `private` method, whereas at line 324, the method is `public` - it is common to group together based on the visibility